samples/appendingEvents.go

```diff
@@ -0,0 +1,142 @@
+package samples
```

Comment:
About file name: in general, you should create single-word file names in Go. But it's not possible sometimes. In that scenario, the snake case is preferred over the camel case.

Example: 
https://github.com/golang/go/tree/master/src/math


![image](https://user-images.githubusercontent.com/5737091/134571494-089fbe8c-3353-4c82-bcc0-c239d0a0491d.png)

Reply:
This module was created only to ease our documentation program so it knows where to parse the snippets

Reply:
ah ok, so we can't do much here, :+1: